src/main/scala/akka/contrib/process/NonBlockingProcess.scala

```diff
@@ -229,7 +245,5 @@ class NonBlockingProcess(
   override def postStop(): Unit = {
     inspectionTick.cancel()
     process.destroy(true)
-    contextMat.shutdown()
```

Comment:
Not required - when the materializer's parent is the current context it will always be shutdown. I personally think that the Akka stream's materializer shutdown method should always be used. Non-actor materializers would probably require the explicit call, and consistency is good. Hindsight is a wonderful thing though.